### ISSUE TITLE
Can't detect rewarded video ad functions on Android at runtime

### ISSUE THREAD

**Issue by grantbridges** [lilili87222/admob-for-flash#75]
I'm having a weird issue with all three of the available ane files when I try to use the rewarded video ad functionality.  I've written code to load a video and everything works fine when I'm testing through my desktop from Adobe Animate (no video is shown, but the code compiles just fine), but when I test it out on my actual Android phone, the apk crashes and the debugger tells me it can't find the "cacheVideo" function.  It's very odd that the code editor seems to "see" the function just fine (see below), but during debugging, Android says the function doesn't exist and the app crashes.  (Also, I tested the isVideoReady() function with the same result, so it seems it can't find any of the video functions.)

![untitled](https://user-images.githubusercontent.com/31935566/31291802-a16057e0-aa96-11e7-88d9-a7c21a185302.png)

**Comment by NathanFitchett** [comment]
Were you able to get the reward video to work?

**Comment by grantbridges** [comment]
No, after wrestling a while I gave up on it. Hard to tell if it's just some weird problem on my end or actually an issue in the ane. Searched online and never found anybody mentioning a similar issue where the function appeared fine in the code editor but couldn't be found at runtime on the android.
